### DISCUSSION TITLE
Where can I download AquaCrop GIS? Target not found.

### DISCUSSION THREAD

**Discussion by 541718883** [KUL-RSDA/AquaCrop#354] [General]
Where can I download AquaCrop GIS? Target not found.

**Comment by gdelannoy** [comment]
The official AquaCrop comes in a 
1. GUI
2. open source code
3. executables for Mac OS, Windows and Linux
4. as part of NASA' Land Information System (LIS).

Spatially distributed **regional AquaCrop simulations** can be obtained by running the AquaCrop executable in a spatial wrapper (https://github.com/KUL-RSDA/RegionalAC_Py), or running AquaCrop inside NASA's LIS (4 above). This output can be visualised in any GIS tool. There is no "GIS" version of AquaCrop publicly available. 

The latest released versions of (1,2,3 above) can be downloaded under "Releases" at the right hand side on https://github.com/KUL-RSDA/AquaCrop. 

AquaCrop in LIS is currently reviewed by the NASA team (4 above), but ready for use here: https://github.com/NASA-LIS/LISF/pull/1633.
